package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "script-solution-reach",
```

el nombre del paquete que sea:
@dathaplus/scripts-solution-reach-sr
```suggestion
  "name": "@dathaplus/scripts-solution-reach-sr",
```